pkg/sql/logictest/testdata/logic_test/upgrade_skip_version

```diff
@@ -1,19 +0,0 @@
-# TODO(radu): reenable this test in 25.2 when we support upgrade from two
-# previous releases (#134455).
-## LogicTest: cockroach-go-testserver-24.2
```

Comment:
shouldn't we be testing an upgrade from 24.2 directly to 25.1 now?

Reply:
No, we won't support that - the minimum supported version is 24.3. We will allow 24.3 to 25.2.

Reply:
it would be nice to keep this file but just have it skipped or disabled somehow so we are reminded to re-add this test.

Reply:
I did keep it, I just renamed it to "upgrade" since it now checks upgrade from all supported versions. So it will automatically run skip upgrades when we support them.
<!-- Reviewable comment -ODqKrcZ7d3RCqkD89nH:b-4xvbww -->
<!-- Sent from Reviewable.io -->

Reply:
ah thanks. github collapsed the file and i would have expected it to identify this as a rename.